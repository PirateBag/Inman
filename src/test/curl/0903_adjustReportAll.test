Report out all adjustements.

=== JSON ===
{ "idToSearchFor" : "-1" }