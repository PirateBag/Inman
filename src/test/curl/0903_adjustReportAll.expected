{
  "responseType" : null,
  "errors" : [ ],
  "data" : [ {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  --------             Adjustment        -------       ----  Item  ----"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Amount    Item   Order   Type       Date  AdTp       Summary    Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "    1.00       1       0   NA    2025-0728  ITEM         W-001       7.00"
  } ]
}